### PR TITLE
docs: Add Airflow Kubernetes pod operator samples

### PR DESCRIPTION
**Pull request by afleisc**

### CODE REVIEW

File: samples/airflow/kubernetes_pod_operator/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+
+### Requirements to run the DAG:
+- Pre-existing Composer v2 environment
+- Build DVT image given below instructions
+
+### Instructions
+
+1. Download the DAG file in this directory
+2. Update the DAF configuration for your use case (update PROJECT_ID, optional gcs bucket)
```
Comment (ajwelch4) [inline samples/airflow/kubernetes_pod_operator/README.md]: typo: `DAF` -> `DAG`

File: samples/airflow/kubernetes_pod_operator/kube_pod_op_dag.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+"""
+
+from datetime import timedelta
+
+import airflow
+from airflow import DAG
+from airflow import models
+from airflow.kubernetes.secret import Secret
```
Comment (ajwelch4) [inline samples/airflow/kubernetes_pod_operator/kube_pod_op_dag.py]: Unused import?

File: samples/airflow/kubernetes_pod_operator/kube_pod_op_dag.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+"""
+
+from datetime import timedelta
+
+import airflow
+from airflow import DAG
+from airflow import models
```
Comment (ajwelch4) [inline samples/airflow/kubernetes_pod_operator/kube_pod_op_dag.py]: Unused import?

**Review by ajwelch4 — APPROVED** [review]
LGTM other than a few ntis.

File: samples/airflow/kubernetes_pod_operator/kube_pod_op_dag.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+)
+
+default_args = {
+    "start_date": airflow.utils.dates.days_ago(1),
+    "retries": 1,
+    "retry_delay": timedelta(minutes=5),
+}
+
+with DAG(
+    "kubernetes_pod_operator",
```
Comment (helensilva14) [inline samples/airflow/kubernetes_pod_operator/kube_pod_op_dag.py]:
```suggestion
    "dvt_dag_kubernetes_pod_operator",
```

**Review by helensilva14 — APPROVED** [review]
LGTM as well, thanks Andrew! +1 to the minor things @ajwelch4 pointed and I added a suggestion for the dag_id

**Pull request merged**